### PR TITLE
br: update readme.md in br

### LINKED ISSUE #51016: Build instructions in br‘s readme.md is outdated
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

build br component as https://github.com/pingcap/tidb/blob/master/br/README.md said

### 2. What did you expect to see? (Required)

successful build after following instructions

### 3. What did you see instead (Required)

root@dev:~/tidb/br# make
make: *** No targets specified and no makefile found.  Stop.

### 4. What is your TiDB version? (Required)

latest

### PR DESCRIPTION
**Pull request by RidRisR**
### What problem does this PR solve?

Issue Number: close #51016

Problem Summary:

the build instruction in br's readme.md is outdated, now update it.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [x] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/README.md [hunk 1]
```diff
@@ -30,7 +30,8 @@
 To build binary and run test:
 
 ```bash
-$ make
+$cd ../tidb
```
Comment (YuJuncen) [inline br/README.md:33]: Or it looks like access the variable `$cd`.
```suggestion
$ cd ../tidb
```

Comment (YuJuncen) [reply br/README.md:33]: (Also, it seems there isn't any example output in this case. Perhaps just removing the dollar will be better?)

**Review by YuJuncen — APPROVED** [review]
Generally LGTM

File: br/README.md (hunk 1 above)
Comment (RidRisR) [reply br/README.md]: maybe, but I see no difference here. As long as no ambiguity will be induse here, I prefer just leave it be.

**Review by BornChanger — APPROVED** [review]

**Pull request merged**